.github/workflows/test-and-deploy.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install wheel # wheel need to be installed before mavproxy
         pip install mavproxy
-        pip install pyfakefs pytest-cov pytest-timeout pylint mypy isort black==22.1.0 asyncmock types-requests \
+        pip install pyfakefs pytest-cov pytest-timeout pylint mypy==0.982 isort black==22.1.0 asyncmock types-requests \
```

Comment:
So, the idea of having not fixed versions is for us to keep improving the code quality and find bugs, as we can see in the patch.

Reply:
so we stop development until mypy fixes the issue?

Reply:
as far as I can tell, this is a mypy issue:
```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.15/x64/bin/mypy", line 8, in <module>
    sys.exit(console_entry())
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/mypy/__main__.py", line 15, in console_entry
    main()
  File "mypy/main.py", line 95, in main
  File "mypy/main.py", line 174, in run_build
  File "mypy/build.py", line 193, in build
  File "mypy/build.py", line 276, in _build
  File "mypy/build.py", line 2903, in dispatch
  File "mypy/build.py", line 3280, in process_graph
  File "mypy/build.py", line 3361, in process_fresh_modules
  File "mypy/build.py", line 2112, in fix_cross_refs
  File "mypy/fixup.py", line 53, in fixup_module
  File "mypy/fixup.py", line 125, in visit_symbol_table
  File "mypy/fixup.py", line 87, in visit_type_info
  File "mypy/types.py", line 1283, in accept
  File "mypy/fixup.py", line 196, in visit_instance
  File "mypy/fixup.py", line 337, in lookup_fully_qualified_typeinfo
AssertionError: Should never get here in normal mode, got Var:ArduPilotManager.Singleton instead of TypeInfo
```